model/RdsResultStorage.php

```diff
@@ -638,6 +643,16 @@ protected function getResultRow($variable)
         return $object;
     }
 
+    /**
+     * Generates a unique, not auto-increment based, primary key.
+     *
+     * @return string
+     */
+    public function getUniquePrimaryKey()
+    {
+        return strrev(uniqid('', true));
```

Comment:
does strrev make uniqid more unique? :)

Reply:
The use of strrev allows use to have non-consecutive uuids (base on currenttime) to be given to spanner as primary key of the table, on which spanner bases the spreading over nodes.
Without the strrev, we would end up with sequenced PK and all would be saved to the same spanner node, creating hotspot and loosing the sharding feature of spanner.

Reply:
applies to bigtable btw also